### PR TITLE
perf: lazy load profile images

### PR DESCRIPTION
**Pull request by Uzlopak**
## Check List

- [ ] I have read the [Contributing Guidelines](./CONTRIBUTING.md)

### CODE REVIEW

**Review by mcollina — APPROVED** [review]
lgtm

File: static/data/team.yml [hunk 1]
```diff
@@ -2,14 +2,14 @@
   people:
     - name: Matteo Collina
       sortname: Collina Matteo
-      picture: 'https://avatars1.githubusercontent.com/u/52195?v=4&s=460'
+      githubId: '52195'
```
Comment (Eomm) [inline static/data/team.yml:7]: it does not seem this change help us IMHO

before:

- copy link
- paste link

now:

- ask yourself: where should I get this id?
- copy link
- paste link
- edit link

Comment (Uzlopak) [reply static/data/team.yml:7]: https://caius.github.io/github_id/#uzlopak

Comment (Eomm) [reply static/data/team.yml:7]: Could you add a comment upper in this file?

Comment (Uzlopak) [reply static/data/team.yml:7]: is it now good?

**Review by Eomm — APPROVED** [review]

**Pull request merged**